web/src/utils/url.ts

```diff
@@ -12,7 +13,11 @@ export const getTimeOfInterestFromUrl = (search: string): string => {
     return queryString
   }
 
-  return formatDateInISO(new Date())
+  const currInPST = moment(new Date())
```

Comment:
```suggestion
  const currentDateInPST = moment()
```
Maybe I'm being pedantic in naming of the variable. But also - you don't need to pass new Date - see https://momentjs.com/docs/#/parsing/now/.

Reply:
huh good to know